### DISCUSSION TITLE
这两天openai的api变得特慢，进官网速度正常。不会是要封我号吧

### DISCUSSION THREAD

**Discussion by heihei2122** [nextai-translator/nextai-translator#603] [Q&A]
这两天openai的api变得特慢，进官网速度正常。不会是要封我号吧

**Comment by fisher-he** [comment]
对，我也是划词翻译好等好几十秒才会出来，什么情况？难不成被限流了？

**Comment by chengfengfengwang** [comment]
+1 一句话要等十几秒才能出来

**Comment by szh-bash** [comment]
前两天不是关了 plus 升级么，当时 gpt4 一直弹公告说部分用户有无响应问题啥的，猜想是 openai 调整了接入服务器限速降低成本以扩大容量服务更多的 User，降低质量提升数量进一步恰$

**Comment by chengfengfengwang** [comment]
家人们有啥好办法同步一下哈，现在这速度根本就不能用，还不如百度翻译

**Comment by heihei2122** [comment]
今天试了一下，速度比昨天快了

**Comment by chengfengfengwang** [comment]
听说gpt 3.5付费的快？

**Comment by usergpt2023** [comment]
If you're interested in learning more about Chat GPT, we recommend reading the article here.
[ChatGPT 가입 방법 및 사용방법](https://kayket.com/it%EB%89%B4%EC%8A%A4/chatgpt-%EA%B0%80%EC%9E%85-%EB%B0%A9%EB%B2%95-%EB%B0%8F-%EC%82%AC%EC%9A%A9%EB%B0%A9%EB%B2%95/)

**Comment by shyetang** [comment]
API付费之后速度一下子就上来了,想办法给api付费吧,反正3个月一到,赠送的就清零了,迟早要付费

**Reply by lamachuang** [reply]
怎么给它付费？PLUS？

**Reply by z1073** [reply]
同问
